src/System.Security.AccessControl/src/System/Security/AccessControl/Win32.cs

```diff
@@ -125,9 +118,7 @@ internal static class Win32
                 {
                     if (handle.IsInvalid)
                     {
-                        throw new ArgumentException(
-                            SR.Argument_InvalidSafeHandle,
-nameof(handle));
+                        throw new ArgumentException(SR.Argument_InvalidSafeHandle, nameof(handle));
```

Comment:
I cleaned the code a bit on purpose in this line so you could see where GetSecurityInfoByName (the P/Invoke method) is called: see line 115 in this file.

Reply:
Related: the other P/Invoke function, GetSecurityInfoByHandle, is called in line 125 in this file.